js/pages/estimation/components/cca-executions.js

```diff
@@ -95,9 +106,31 @@ define([
 		}
 
 		dispose() {
+			super.dispose();
 			PollService.stop(this.pollId);
 		}
 
+		highlightGeneration() {
+			if (this.isSourceListLoaded()) {
+				if (this.notificationSourceId() && this.notificationExecutionId()) {
```

Comment:
Why nested if-s? Cannot we squash all the conditions into a single if statement?

Reply:
Fixed